bootloader/Dockerfile

```diff
@@ -44,19 +42,19 @@ WORKDIR /home/
 RUN wget https://zlib.net/zlib-${ZLIB_VERSION}.tar.gz \
     && tar xf zlib-$ZLIB_VERSION.tar.gz && rm zlib-$ZLIB_VERSION.tar.gz
 
+ENV CC=${CC:-gcc}
+
 WORKDIR zlib-$ZLIB_VERSION
 # Install zlib-devel but in the sysroot of the cross compiler.
 RUN ./configure --prefix="$($CC -print-sysroot)"
 RUN make
 RUN make install
 
 WORKDIR /io
-# Redefine the C compiler so that works.
-# Because this compiler is so old, C99 features need to be explicitly enabled
-# and PATH_MAX needs to be manually defined. -std=c99 truly means strict
-# posix C99 compliance which disables the non-posix symbols like seeko() which
-# we use for huge files. The last define re-enables them.
+# Redefine the C compiler so that works. Because this compiler is so old, C99 features need to be explicitly enabled and
+# PATH_MAX needs to be manually defined. -std=c99 truly means strict posix C99 compliance which disables the non-posix
+# symbols like seeko() which we use for huge files. The last define re-enables them.
```

Comment:
Why do we need to enable C99?

Reply:
That's from before #6086 so it's obsolete now. I suppose we might as well get rid of it.

Reply:
And even if we did need it, it could have all been replaced with `-gnu99` anyway - I just didn't understand what the `-gnuxx` variants were back then.